lib/patches/db/pg.rb

```diff
@@ -39,6 +39,19 @@ class PG::Connection
     alias_method :exec_params_without_profiling, :exec_params
   end
 
+  def prepare_parameterized_statement(args)
+    if Array === args[1] && args[1].length > 0
+      query = args[0]
+      parameters = args[1]
+      counter = 0
+      loop do
+        break if !query.include? "$#{counter+=1}"
+        query = query.sub("$#{counter}",parameters[counter-1].to_s)
```

Comment:
subbing is a bit confusing and can cause a nonsense query to be logged in some cases. For example: 

$1 -> hello world's, which technically would need to be `'hello world''s'` I think

Reply:
an alternative may be a preamble comment that explains the param so there is not confusion.  also I think you technically want gsub here, also what if more than 10 params? 

`$1` -> `\* $1 *\ 'hello-world''s'`

Reply:
What will happen if more than 10 params?